src/sidebar/tabcenter.css

```diff
@@ -680,17 +699,19 @@ body[platform="win"] #searchbox-input:placeholder-shown {
 .tab-close {
```

Comment:
Almost pixel perfect, but the size of the **X** was bothering me:

![comp](https://user-images.githubusercontent.com/6424575/40318485-ccd001b4-5cf2-11e8-972d-8a293f1459e9.png)
The 40x40px gray hover background is the right size however!